src/screens/FeedScreen/Header.js

```diff
@@ -456,21 +456,21 @@ const styles = StyleSheet.create({
   feedDate: {
     fontFamily: fonts.inter[400],
     fontSize: 12,
-    color: colors.blackgrey,
+    color: COLORS.blackgrey,
     lineHeight: 18
   },
   feedDateLocation: {
     flex: 1,
     fontFamily: fonts.inter[400],
     fontSize: 12,
-    color: colors.blackgrey,
+    color: COLORS.blackgrey,
     lineHeight: 18
   },
   point: {
     width: 2,
     height: 2,
     borderRadius: 4,
-    backgroundColor: colors.gray,
+    backgroundColor: COLORS.gray8,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color values have been updated from `colors.blackgrey` and `colors.gray` to `COLORS.blackgrey` and `COLORS.gray8` respectively. These changes are consistent with the update in the import statement.
```diff
- color: colors.blackgrey,
+ color: COLORS.blackgrey,
...
- backgroundColor: colors.gray,
+ backgroundColor: COLORS.gray8,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->